Index: src/Second_practice/TestDog.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package Second_practice;\r\nimport java.util.Scanner;\r\n\r\npublic class TestDog {\r\n    public static void main(String[] args) {\r\n        Dog[] list = new Dog[10];\r\n        Scanner sc = new Scanner(System.in);\r\n        System.out.println(\"Enter the number of dogs\");\r\n        int j = sc.nextInt();\r\n        sc.nextLine();\r\n        String tempName;\r\n        int tempAge;\r\n        for(int i = 0; i < j; i++){\r\n            System.out.println(\"Enter the dog's name\");\r\n            tempName = sc.nextLine();\r\n            System.out.println(\"Enter the dog's age\");\r\n            tempAge = sc.nextInt();\r\n            list[i] = new Dog(tempName, tempAge);\r\n            sc.nextLine();\r\n        }\r\n        System.out.println(\"Age of dog \"+ list[0].getName() + \" in human counting: \" + list[0].toHumanAge());\r\n    }\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Second_practice/TestDog.java b/src/Second_practice/TestDog.java
--- a/src/Second_practice/TestDog.java	(revision 487cace334431fb7f3ebd7e675bd440c802a31ef)
+++ b/src/Second_practice/TestDog.java	(date 1662487443705)
@@ -3,14 +3,14 @@
 
 public class TestDog {
     public static void main(String[] args) {
-        Dog[] list = new Dog[10];
         Scanner sc = new Scanner(System.in);
         System.out.println("Enter the number of dogs");
-        int j = sc.nextInt();
+        int listLength = sc.nextInt();
+        Dog[] list = new Dog[listLength];
         sc.nextLine();
         String tempName;
         int tempAge;
-        for(int i = 0; i < j; i++){
+        for(int i = 0; i < listLength; i++){
             System.out.println("Enter the dog's name");
             tempName = sc.nextLine();
             System.out.println("Enter the dog's age");
